database/101-up-postgres/20231211120959-solana_winner_tracking_add_intermediate_winning_hash.sql

```diff
@@ -0,0 +1,12 @@
+
+-- migrate:up
+
+CREATE TABLE solana_intermediate_winners (
+	created TIMESTAMP NOT NULL DEFAULT CURRENT_TIMESTAMP,
+	winning_signature VARCHAR NOT NULL,
+	payout_signature VARCHAR NOT NULL
```

Comment:
these should be consistent with their names in `winners`, or at least commented as such